### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by Chr1stian**

### CODE REVIEW

File: my-app/src/app/app.component.spec.ts [hunk 1]
```diff
@@ -1,10 +1,132 @@
...
+import { PageNotFoundComponent } from './page-not-found/page-not-found.component';
+import { WineSearchComponent } from './wine-search/wine-search.component';
+import { LoginComponent } from './login/login.component';
+import { RegisterComponent } from './register/register.component';
+import { LoginDialogComponent } from './login-dialog/login-dialog.component';
+import { MapComponent } from './map/map.component';
+import { SingleWineComponent } from './single-wine/single-wine.component';
+import { ProfileComponent } from './profile/profile.component';
+import { UsersWinesComponent } from './users-wines/users-wines.component';
+import {BrowserAnimationsModule} from '@angular/platform-browser/animations';
```
Comment (emilps) [inline my-app/src/app/app.component.spec.ts]: Missing spaces

File: my-app/src/app/app.component.spec.ts [hunk 2]
```diff
@@ -1,10 +1,132 @@
...
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+
+import { APP_BASE_HREF } from '@angular/common';
+
+const appRoutes: Routes = [
+  {
+    path: '',
+    component: WineSearchComponent,
+  },
+
```
Comment (emilps) [inline my-app/src/app/app.component.spec.ts]: Remove space

File: my-app/src/app/app.component.spec.ts [hunk 3]
```diff
@@ -1,10 +1,132 @@
...
+  {
+    path: 'navbar',
+    component: NavbarComponent,
+    canActivate: [AuthGuard]
+    //Not full functional yet. Make it async maybe?
+  },
+
```
Comment (emilps) [inline my-app/src/app/app.component.spec.ts]: Remove

File: my-app/src/app/login-dialog/login-dialog.component.spec.ts [hunk 4]
```diff
@@ -1,14 +1,122 @@
...
+
+class MdDialogRefMock {
+}
+
+const appRoutes: Routes = [
+  {
+    path: '',
+    component: WineSearchComponent,
+  },
+
```
Comment (emilps) [inline my-app/src/app/login-dialog/login-dialog.component.spec.ts]: remove space

File: my-app/src/app/login-dialog/login-dialog.component.spec.ts [hunk 5]
```diff
@@ -1,14 +1,122 @@
...
+  {
+    path: 'navbar',
+    component: NavbarComponent,
+    canActivate: [AuthGuard]
+    //Not full functional yet. Make it async maybe?
+  },
+
```
Comment (emilps) [inline my-app/src/app/login-dialog/login-dialog.component.spec.ts]: remove newline

File: my-app/src/app/navbar/navbar.component.spec.ts [hunk 6]
```diff
@@ -1,14 +1,123 @@
...
+
+class MdDialogRefMock {
+}
+
+const appRoutes: Routes = [
+  {
+    path: '',
+    component: WineSearchComponent,
+  },
+
```
Comment (emilps) [inline my-app/src/app/navbar/navbar.component.spec.ts]: fix newlines

File: my-app/src/app/register/register.component.spec.ts [hunk 7]
```diff
@@ -1,14 +1,124 @@
...
+
+class MdDialogRefMock {
+}
+
+const appRoutes: Routes = [
+  {
+    path: '',
+    component: WineSearchComponent,
+  },
+
```
Comment (emilps) [inline my-app/src/app/register/register.component.spec.ts]: same here

File: my-app/src/app/wine-search/wine-search.component.spec.ts [hunk 8]
```diff
@@ -1,25 +1,61 @@
...
 
+import { HttpModule } from '@angular/http';
+import { FavoriteWineService } from './../services/favoritewine.service';
+import { UserService } from './../services/users.service';
+import { SingleWineComponent } from './../single-wine/single-wine.component';
+import { Subscription } from 'rxjs/Subscription';
+import { MessageService } from './../services/message.service';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { DataService } from './../data.service';
+import {BrowserAnimationsModule} from '@angular/platform-browser/animations';
```
Comment (emilps) [inline my-app/src/app/wine-search/wine-search.component.spec.ts]: Kevin spacey here

**Comment by Chr1stian** [comment]
Updated to match your changes on master
@emilps

**Pull request merged**